yt/yt/server/queue_agent/queue_static_table_exporter.cpp

```diff
@@ -160,7 +189,7 @@ class TQueueExportTask
             .ValueOrThrow()
             .NodeId;
 
-        // Take it for guarantee that only one queue agent instance do this export.
+        // Take it for granted that only one queue agent instance do this export.
```

Comment:
It is not possible in a sense that there can't be 2 concurrent transactions doing the same export since we take the same shared lock. It might be possible if leader switch and config change occurred at the same time, but I don't think that is a problem.
Do you agree with that? Or am I missing something?
If you do agree, then I'll re-write the comment to reflect this behavior.